OrbitGl/TimeGraph.cpp

```diff
@@ -848,6 +935,44 @@ std::shared_ptr<GpuTrack> TimeGraph::GetOrCreateGpuTrack(
   return track;
 }
 
+//-----------------------------------------------------------------------------
+void SetTrackNameFromRemoteMemory(std::shared_ptr<Track> track,
+                                  uint64_t string_address) {
+  const auto& thread_pool = GOrbitApp->GetThreadPool();
```

Comment:
consider passing executor/thread pool here instead of using global.

Reply:
or OrbitApp* app

Reply:
Done.